lib/action_subscriber/router.rb

```diff
@@ -58,7 +58,11 @@ def local_application_name
                                  when ENV['APP_NAME'] then
                                    ENV['APP_NAME'].to_s.dup
                                  when defined?(::Rails) then
-                                   ::Rails.application.class.parent_name.dup
+                                   if ::Rails.application.class.respond_to?("module_parent_name")
```

Comment:
Nit: respond_to? should probably use a symbol arg.

Reply:
Good idea. Fixed.